libs/coin-evm/src/prepareTransaction.ts

```diff
@@ -28,7 +28,7 @@ export const prepareCoinTransaction = async (
     const gasLimit = await getGasEstimation(account, {
       ...typedTransaction,
       amount: new BigNumber(0),
-    });
+    }).catch(() => new BigNumber(0)); // this catch returning 0 should be handled by the `getTransactionStatus` method
```

Comment:
`getTransactionStatus` does not seem to call `prepareCoinTransaction`

https://github.com/LedgerHQ/ledger-live/blob/feat/add-telos-currency/libs/coin-evm/src/getTransactionStatus.ts#L194-L233

Reply:
`getTransactionStatus` is called after `prepareTransaction` in the bridge so it's normal